pyproject.toml

```diff
@@ -3,11 +3,14 @@ requires = ["setuptools>=61.2"]
 build-backend = "setuptools.build_meta"
 
 [project]
-name = "pycaravel"
```

Comment:
I think you want the package name to be `caravel`. Why not rename this repository `pycaravel` → `caravel`?